x-pack/plugins/enterprise_search/server/lib/enterprise_search_request_handler.test.ts

```diff
@@ -378,6 +404,34 @@ describe('EnterpriseSearchRequestHandler', () => {
     });
   });
 
+  describe('setSessionData', () => {
+    it('sets the value of wsOAuthTokenPackage in a cookie', async () => {
+      const mockNow = 'Thu, 04 Mar 2021 22:40:32 GMT';
+      const mockInAnHour = 'Thu, 04 Mar 2021 23:40:32 GMT';
+      const tokenPackage = 'sEkReTZ';
+
+      const expectedCookie = `${ENTERPRISE_SEARCH_KIBANA_COOKIE}=${tokenPackage}; Path=/; Expires=${mockInAnHour}; SameSite=Lax; HttpOnly`;
```

Comment:
This feels a little odd to me being declared so far from where it's actually asserted on. I'd suggest either not abstracting `expectedCookie` out to a var (something I generally avoid during unit tests - it makes it easier to read/skim a test quickly without having to bounce up and down back up the test to check real values or have to hold values mentally in your head), or declaring `expectedCookie` right before the `expect(enterpriseSearchRequestHandler.headers).toEqual({` assertion at the bottom of the test.